etcd-manager/cmd/etcd-manager/main.go

```diff
@@ -161,6 +163,9 @@ type EtcdManagerOptions struct {
 
 	// EtcdManagerMetricsPort allows exposing statistics from etcd-manager
 	EtcdManagerMetricsPort int
+
+	// IPFilter allows filtering for IP addresses with CIDR notation
+	IPFilter string
```

Comment:
Seems like a good idea to me, just not sure about the name of the option. `NetworkCIDR` or `ClusterCIDR` may be closer to what kOps uses.
Also, in some cases there could be additional CIDRs, so multiple ones should be accepted.

What do you think @justinsb?

Reply:
I agree, the naming is definitely not optimal, I'm open for suggestions. :)

To allow multiple opts I can maybe do smthg like this: https://github.com/kubernetes/cloud-provider-openstack/pull/1946

Either comma separated, or allow multiple cli opts, and priortize by order ("first come first serve").

Reply:
@hakman I have updated the name to `NetworkCIDR` and it is now possible to specify multiple, comma separated CIDRs